include.js

```diff
@@ -6,6 +6,15 @@ let cards = [
     author: 'Ankit',
     githubLink: 'https://github.com/ankit0049'
   },
+  let cards = [
+  {
+    artName: 'Moonrise', 
+    pageLink: './Art/ahmedalhamad7/moon.html', 
+    imageLink: './Art/ahmedalhamad7/static moon.png', 
+    author: 'Ahmed', // use your name
+    githubLink: 'https://github.com/ahmedalhamad7'
+  }
+];
```

Comment:
This will break the code.
Please remove line 9 & line 17.

Please fix this.

Reply:
I did the changes as requested, sorry for the mistake, and thanks for notifying.